src/matrix.jl

```diff
@@ -165,28 +198,66 @@ function ∈(M::AbstractMatrix, A::AbstractIntervalMatrix)
     return true
 end
 
+# random interval
+function rand(::Type{Interval}; N::Type{<:Real}=Float64,
+              rng::AbstractRNG=GLOBAL_RNG)::Interval{N}
+    x, y = randn(rng, N), randn(rng, N)
+    return x < y ? Interval(x, y) : Interval(y, x)
+end
+
 """
-    rand(A::IntervalMatrix{T}; rng::AbstractRNG=GLOBAL_RNG) where {T}
+    rand(::Type{IntervalMatrix}, m::Int=2, n::Int=2;
+         N=Float64, rng::AbstractRNG=GLOBAL_RNG)
 
-Sample a concrete matrix ``B`` from an interval matrix ``A`` such that every
-entry in ``B`` belongs to the corresponding interval in ``A``.
+Return a random interval matrix of the given size and numeric type.
 
 ### Input
 
-- `A`   -- interval matrix
-- `rng` -- (optional, default: `GLOBAL_RNG`) random-number generator
+- `IntervalMatrix` -- type, used for dispatch
+- `m`              -- (optional, default: `2`) number of rows
+- `n`              -- (optional, default: `2`) number of columns
+- `rng`            -- (optional, default: `GLOBAL_RNG`) random-number generator
+
+### Output
+
+An interval matrix of size ``m × n`` whose coefficients are normally-distributed
+intervals of type `N` with mean `0` and standard deviation `1`.
+"""
+function rand(::Type{IntervalMatrix}, m::Int=2, n::Int=2;
+              N=Float64, rng::AbstractRNG=GLOBAL_RNG)
+    B = Matrix{Interval{N}}(undef, m, n)
+    for j in 1:n
+        for i in 1:m
+            @inbounds B[i, j] = rand(Interval, N=N)
+        end
+    end
+    return IntervalMatrix(B)
+end
+
+"""
+    sample(A::IntervalMatrix{T}; rng::AbstractRNG=GLOBAL_RNG) where {T}
```

Comment:
In LazySets, `rand` works with types and `samples` works with sets. Here it is the same.

Reply:
Then we should change it in `LazySets`!

Reply:
I checked `LazySets`: We only have one such method for `Ball2` and there is an `nsamples` argument. With that I find it acceptable to use a different function name because this is not the standard for `rand`. But here we only return one matrix, which is consistent with `rand`.

Reply:
The default implementation for `rand(A::AbstractMatrix)` returns a random entry of `A`. In favor of not breaking this rule, I agree to use `sample` here.